utils/tolerance.stanza

```diff
@@ -0,0 +1,65 @@
+#use-added-syntax(jitx)
+defpackage ocdb/tolerance:
+  import core
+  import collections
+  import math
+
+; A generic value with some tolerance. May be offcenter (upper tolerance != lower tolerance)
+public pcb-struct ocdb/tolerance/Toleranced:
+  typ: Double,
+  tol+:Double,
+  tol-:Double,
+
+defmethod print (o:OutputStream, t:Toleranced):
```

Comment:
add in unit test, not as `print` but as `to-string`, and call `#EXPECT` on the result

Reply:
done